lib/utils/azure_storage.ts

```diff
@@ -0,0 +1,48 @@
+import * as azureStorage from "azure-storage";
+import { Option, option } from "ts-option";
+import { Either, left, right } from "./either";
+
+// Storage connection string may be process.env.AzureWebJobsStorage;
+
+export type BlobService = azureStorage.BlobService;
+export type BlobResult = azureStorage.BlobService.BlobResult;
+
+export function getBlobService(connectionString: string): BlobService {
+  return azureStorage.createBlobService(connectionString);
+}
+
+export function getBlobUrl(
+  blobService: BlobService,
+  containerName: string,
+  attachmentName: string
+): string {
+  return blobService.getUrl(containerName, attachmentName);
```

Comment:
ditto!

helper functions should add some value, be convenient in some way!

Reply:
see https://github.com/teamdigitale/digital-citizenship-functions/pull/59#discussion_r141296878